src/common/utils/NebulaKeyUtils.h

```diff
@@ -251,6 +258,108 @@ class NebulaKeyUtils final {
         return rawKey.subpiece(0, rawKey.size() - sizeof(int64_t));
     }
 
+    // Only int and double are supported
+    static std::string boundVariant(nebula::cpp2::SupportedType type,
+                                    NebulaBoundValueType op,
+                                    const VariantType& v = 0L) {
+        switch (op) {
+            case NebulaBoundValueType::kMax : {
+                std::vector<unsigned char> bytes = {0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF};
+                return std::string(bytes.begin(), bytes.end());
+            }
+            case NebulaBoundValueType::kMin : {
+                std::string str;
+                str.reserve(sizeof(int64_t));
+                str.append(sizeof(int64_t), '\0');
+                return str;
```

Comment:
static std::string str("\0\0\0\0\0\0\0\0\0\0\0", sizeof(int64_t)); return str;

Reply:
> static std::string str("\0\0\0\0\0\0\0\0\0\0\0", sizeof(int64_t)); return str;

What's the difference?

Reply:
less a constructor function about string call and memory allocate and copy